pkgs/by-name/op/openmpi/package.nix

```
@@ -176,12 +178,14 @@ stdenv.mkDerivation (finalAttrs: {
           # compiler=_ line that should be replaced by a compiler=#2 string, where
           # #2 is the 2nd value in the list.
           "cc" = [
-            "gcc"
-            "${targetPackages.stdenv.cc}/bin/${targetPackages.stdenv.cc.targetPrefix}cc"
+            # "$CC" is equivalent to ${"clang" if stdenv.hostPlatform.isDarwin else "gcc"}
```

That's somewhat correct but a very confusing comment! How about:

```suggestion
            # "$CC" is expanded by the executing shell in the substituteInPlace
            # commands to the name of the compiler ("clang" for Darwin and
            # "gcc" for Linux)
```